setup.cfg

```
@@ -25,37 +25,39 @@ install_requires =
 test =
     pytest-astropy
     pytest-cov
+    regions>=0.7
+    numpy>=1.24.0
+    astropy>=5.2.1
 docs =
     sphinx-astropy
     matplotlib
-novis =
+noviz =
     zarr
     fsspec
     distributed
     pvextractor
-    regions>=0.3
-    reproject
+    reproject>=0.9.1
     scipy
-all =
-    pytest-astropy
-    pytest-cov
-    sphinx-astropy
-    zarr
-    fsspec
-    distributed
+viz =
     aplpy
-    glue-core[qt]
     matplotlib
-    pvextractor
-    regions>=0.3
     reproject
-    scipy
+    pvextractor
+viz_extra =
+    glue-core[qt]
     yt ; python_version<'3.8'
+dev =
+    pvextractor @ git+https://github.com/radio-astro-tools/pvextractor#egg=pvextractor
+    radio-beam @ git+https://github.com/radio-astro-tools/radio-beam#egg=radio-beam
+    astropy @ git+https://github.com/astropy/astropy#egg=astropy
+    #reproject @ git+https://github.com/astropy/reproject#egg=reproject
+    reproject @ git+https://github.com/keflavich/python-reprojection.git@high_level_wcs#egg=reproject
```

We should revert this line before merging, i.e.:
```suggestion
    reproject @ git+https://github.com/astropy/reproject#egg=reproject
```

but we need https://github.com/astropy/reproject/pull/349 merged first

---

reverted.